webrtc/services/src/main/java/org/exoplatform/webconferencing/webrtc/rest/RESTWebRTCService.java

```diff
@@ -188,7 +188,7 @@ public Response getSettings(@Context UriInfo uriInfo) {
   }
 
   @GET
-  @RolesAllowed("administrators")
+  @RolesAllowed("users")
```

Comment:
Could you explain why this service must be readable by users ?
As it exposes some informations like internal urls or credentials, I prefer that it stay limited to administrators

Reply:
I will revert this as I won't use the Rest endpoint. But those same settings are exposed by the JS initiation of web conferencing connector and could be gotten from there too like in https://github.com/exoplatform/web-conferencing/blob/develop/webrtc/webapp/src/main/java/org/exoplatform/webconferencing/webrtc/portlet/WebrtcProviderPortlet.java#L66
We may consider to improve this later WDYT ?

Reply:
I dont know if a simple user have know it or not.
In fact, as we will remove webrtc, and keep only jitsi, it is not huge priority.
So, please create a task somewhere to check it for jitsi, and yes we will consider it later